addon/components/input-validator.js

```diff
@@ -47,10 +47,14 @@ export default Component.extend({
         return set(this, 'hasFocusedOut', true);
     },
 
-    didRender() {
+    didInsertElement() {
```

Comment:
@billdami did you ensure that this works in both fastboot/out of fastboot? I did this so long ago, but I remember it had to be didRender for a reason

Reply:
@bakerac4 I didnt specifically test in fastboot, but `didInsertElement` is the recommended hook to use for DOM-accessing code as its guaranteed not to run in fastboot.

Reply:
@bakerac4 i reverted this back to `didRender` just to lessen the chance of any regressions in apps that were depending on it using that lifecycle hook. Though we might want to revisit this in the future, as `didInsertElement` is the safer route I think.